client/analytics/settings/index.js

```diff
@@ -50,6 +60,18 @@ class Settings extends Component {
 		/* eslint-enable no-console */
 	}
 
+	warnIfUnsavedChanges( event ) {
+		const { isDirty } = this.state;
+
+		if ( isDirty ) {
+			event.returnValue = __(
+				'You have unsaved changes. If you proceed, they will be lost.',
+				'woocommerce-admin'
+			);
+			return event.returnValue;
```

Comment:
Where does this string get rendered?

Chrome
![Screen Shot 2019-07-01 at 1 05 16 PM](https://user-images.githubusercontent.com/1922453/60404955-5535cb80-9c01-11e9-96ff-4c80e20dcbbe.png)

FF
![Screen Shot 2019-07-01 at 1 07 56 PM](https://user-images.githubusercontent.com/1922453/60404957-59fa7f80-9c01-11e9-89dc-3115be732f1d.png)

Reply:
Those are built into the browsers [#](https://stackoverflow.com/questions/51681574/how-to-get-the-leave-site-changes-that-you-made-may-not-be-saved-when-trying/51681800)